README.rst

```diff
@@ -1196,6 +1196,11 @@ Choose from the list of available rules:
   When making a method or function call, there MUST NOT be a space between
   the method or function name and the opening parenthesis.
 
+  Configuration options:
+
+  - ``fix_special_constructs`` (``bool``): whether to fix ``echo``, ``print``,
```

Comment:
The name `fix_special_constructs` is really confusing - why these 6 are "special" and other from `$tokenKinds` are not?

Can we have `tokens` option like in `no_extra_blank_lines`?

Reply:
`array`, `empty` etc. are language constructs (dedicated tokens, cannot be called using [variable functions](https://www.php.net/manual/en/functions.variable-functions.php)) but require parentheses around their argument(s) (just like regular functions), whereas
`echo`, `print` etc. are "special" [language] constructs in that they don't expect parentheses around their argument(s) (like `return` or `throw`).
That's maybe confusing (and I could even add "why not `clone`?") but that's how the fixer was created (and see #1266) and my attempts to fix it so far have been blocked or ignored (at least that's how it feels like) 😕

Thanks for the review, I shall try to do like `NoExtraBlankLinesFixer` when I get more time...